scripts/yocto-build-and-test.sh

```diff
@@ -212,6 +212,10 @@ EOF
         cat >> $BUILDDIR/conf/local.conf <<EOF
 PREFERRED_PROVIDER_virtual/mender-client = "mender"
 PREFERRED_PROVIDER_virtual/mender-client-native = "mender-native"
+EOF
+        # Remove after MEN-6279
+        cat >> $BUILDDIR/conf/local.conf <<EOF
+PREFERRED_VERSION:pn-mender-flash = "master-git%"
```

Comment:
Maybe an explicit mention in the ticket is useful? This probably will keep working even if we add a recipe, so it could be forgotten.

Reply:
Exactly. I was planning to do that.